File: homework_6/calculations/Division.java
package calculations;

//класс для нахождения частного двух чисел
public class Division implements I__Calculator{
    private double a;
    private double b;

    public Division(double a, double b){
        this.a = a;
        this.b = b;
    }

    @Override
    public double getResult(){
        return a / b;
    }
}

File: homework_7/calculations/division.py
from calculations.I__calculator import Calculations

"""наследуем класс Division от базового Calculations"""
class Division(Calculations):
    def __init__(self, a, b):
        self.a = a
        self.b = b
    
    """переопределяем абстрактый метод"""
    def get_result(self):
        return self.a / self.b